### PR TITLE
isContentEditable refactored (connected with #1366)

### PR DESCRIPTION
**Pull request by helen-dikareva**

### CODE REVIEW

**Comment by miherlosev** [comment]
ERRORS:

Linux android 5.1 - /fixtures/utils/dom-test.js

Test: isContentEditableElement

message: failed, expected argument to be truthy, was: false

expected: true

actual: false

File: test/client/fixtures/utils/dom-test.js [hunk 1]
```diff
@@ -364,6 +364,80 @@ test('isHammerheadAttr', function () {
...
+
+    document.designMode = 'on';
+    ok(domUtils.isContentEditableElement(document));
+    ok(domUtils.isContentEditableElement(document.body));
+    document.designMode = 'off';
+
+    // isRenderedNode
+    /* eslint-disable no-empty */
+    try {
+        var processingInstruction = document.createProcessingInstruction('xml-stylesheet', 'href="mycss.css" type="text/css"');
```
Comment (miherlosev) [inline test/client/fixtures/utils/dom-test.js]: Does `try...catch` connect with Android 5.1?
Add comment why we need try/catch there.

**Comment by helen-dikareva** [comment]
FPR

File: test/client/fixtures/utils/dom-test.js (hunk 1 above)
Comment (helen-dikareva) [reply test/client/fixtures/utils/dom-test.js]: in IE and Android `document.createProcessingInstruction` raises an error, ok I''ll add comment

Comment (LavrovArtem) [reply test/client/fixtures/utils/dom-test.js]: https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/utils/dom-test.js#L131-L132

Comment (helen-dikareva) [reply test/client/fixtures/utils/dom-test.js]: ok, thanks

**Comment by helen-dikareva** [comment]
FPR

**Pull request merged**